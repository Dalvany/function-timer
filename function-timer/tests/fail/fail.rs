use function_timer::time;

struct Test {}

impl Test {
    #[time(struct)]
    fn test() {}
}
