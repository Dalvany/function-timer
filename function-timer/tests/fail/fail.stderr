error: Expected literal or identifier
 --> tests/fail/fail.rs:6:12
  |
6 |     #[time(struct)]
  |            ^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/fail/fail.rs:8:2
  |
8 | }
  |  ^ consider adding a `main` function to `$DIR/tests/fail/fail.rs`
